src/calculateRentalCost.js

```diff
@@ -4,7 +4,17 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
```

Comment:
Consider renaming the parameter `days` to `numberOfDays` for better readability and to make the purpose of the variable clearer.